src/compojure/api/logging.clj

```diff
@@ -0,0 +1,20 @@
+(ns compojure.api.logging
```

Comment:
Is this namespace supposed to be used just by compojure-api itself or other users also? Would be good to mention here.